### ISSUE TITLE
[FFmpeg] Cannot enconde with `libvpx-vp9`: Failed to initialize encoder: ABI version mismatch

### ISSUE THREAD

**Issue by thesword53** [msys2/MINGW-packages#22914]
### Description / Steps to reproduce the issue

`ffmpeg -i input.mp4 -an -c:v libvpx-vp9 output.mp4`

### Expected behavior

Video encode working.

### Actual behavior

```
[libvpx-vp9 @ 000002414af6f640] v1.15.0
[libvpx-vp9 @ 000002414af6f640] vpx_codec_enc_cfg
[libvpx-vp9 @ 000002414af6f640] generic settings
  g_usage:                      0
  g_threads:                    16
  g_profile:                    1
  g_w:                          2560
  g_h:                          1440
  g_bit_depth:                  8
  g_input_bit_depth:            8
  g_timebase:                   {1/180}
  g_error_resilient:            0
  g_pass:                       0
  g_lag_in_frames:              25
[libvpx-vp9 @ 000002414af6f640] rate control settings
  rc_dropframe_thresh:          0
  rc_resize_allowed:            0
  rc_resize_up_thresh:          60
  rc_resize_down_thresh:        30
  rc_end_usage:                 3
  rc_twopass_stats_in:          0000000000000000(0)
  rc_target_bitrate:            256
[libvpx-vp9 @ 000002414af6f640] quantizer settings
  rc_min_quantizer:             0
  rc_max_quantizer:             63
[libvpx-vp9 @ 000002414af6f640] bitrate tolerance
  rc_undershoot_pct:            25
  rc_overshoot_pct:             25
[libvpx-vp9 @ 000002414af6f640] temporal layering settings
  ts_number_layers:             1
[libvpx-vp9 @ 000002414af6f640]
  layer_target_bitrate:         0 0 0 0 0
[libvpx-vp9 @ 000002414af6f640]
  ts_rate_decimator:            0 0 0 0 0
[libvpx-vp9 @ 000002414af6f640]
  ts_periodicity:               0
[libvpx-vp9 @ 000002414af6f640]
  ts_layer_id:                  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
[libvpx-vp9 @ 000002414af6f640] decoder buffer model
  rc_buf_sz:                    6000
  rc_buf_initial_sz:            4000
  rc_buf_optimal_sz:            5000
[libvpx-vp9 @ 000002414af6f640] 2 pass rate control settings
  rc_2pass_vbr_bias_pct:        50
  rc_2pass_vbr_minsection_pct:  0
  rc_2pass_vbr_maxsection_pct:  2000
[libvpx-vp9 @ 000002414af6f640]   rc_2pass_vbr_corpus_complexity:0
[libvpx-vp9 @ 000002414af6f640] keyframing settings
  kf_mode:                      1
  kf_min_dist:                  0
  kf_max_dist:                  128
[libvpx-vp9 @ 000002414af6f640]
[libvpx-vp9 @ 000002414af6f640] Failed to initialize encoder: ABI version mismatch
[vost#0:0/libvpx-vp9 @ 000002414af6efc0] Error while opening encoder - maybe incorrect parameters such as bit_rate, rate, width or height.
[vf#0:0 @ 0000024149346600] Error sending frames to consumers: Invalid argument
[vf#0:0 @ 0000024149346600] Task finished with error code: -22 (Invalid argument)
[vf#0:0 @ 0000024149346600] Terminating thread with return code -22 (Invalid argument)
[vost#0:0/libvpx-vp9 @ 000002414af6efc0] Could not open encoder before EOF
[vost#0:0/libvpx-vp9 @ 000002414af6efc0] Task finished with error code: -22 (Invalid argument)
[vost#0:0/libvpx-vp9 @ 000002414af6efc0] Terminating thread with return code -22 (Invalid argument)
[out#0/mp4 @ 0000024149404440] Nothing was written into output file, because at least one of its streams received no packets.
frame=    0 fps=0.0 q=0.0 Lsize=       0KiB time=N/A bitrate=N/A speed=N/A
Conversion failed!
```

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-19045

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [X] UCRT64
- [ ] CLANG64
- [ ] CLANGARM64

**Comment by lazka** [comment]
weird, there was no ABI change with the recent vpx update

**Comment by Biswa96** [comment]
There was an ABI change but release notes did not mention it. See https://gitlab.archlinux.org/archlinux/packaging/packages/ffmpeg/-/issues/14 and https://gitlab.archlinux.org/archlinux/packaging/packages/ffmpeg/-/commit/06e1acba59048538b32b839c781ea153e9c2fb07

**Comment by lazka** [comment]
Fixed via cf826b4d095ad16d013a9da1c7f1a849579657aa

thanks for the report
